mite_cli.py

```diff
@@ -234,6 +242,45 @@ def edit(mite, id, date, minutes, project_id, service_id, note):
 
     print("Entry edited for {}!".format(date))
 
+# from-date and to-date are strings, so I think its okay to support
+# special keys such as yesterday and today.
+@cli.command()
+@click.option("--from-date", default="yesterday", help="From date"
+              " (default is today)")
+@click.option("--to-date", default="today", help="To date"
+              " (default is yesterday)")
+@click.pass_obj
+def replicate(mite, from_date, to_date):
+    '''Replicate entries from a day to another.
+
+    Date parameters accept date either in YYYY-MM-DD format or one of
+    the special keywords (yesterday, today, tomorrow)
+    '''
+    f_dt = parse_date(from_date)
+    t_dt = parse_date(to_date)
+
+    print("Replicating entries from {} to {}".format(f_dt, t_dt))
+
+    # from=f_dt obviously won't fly, but is there a better way to pass
+    # this?
+    entries = mite.list_entries(**{"from": f_dt, "to": f_dt})
```

Comment:
I don’t think I quite understand. This will filter all entries from a day to the target date, and then replicate all of them. Would it not make more sense to take the starting date, add 24 hours, and then replicate those to the target date?

Reply:
Is there a difference between this query and date+24h in mite? The comment is referencing to the fact you can't pass `from=foo` as an argument to list entries because following is a syntax error

```
def foo(**kwargs):
    print(kwargs)

foo(from="what")
```

Reply:
The problem is the following: if I want to port my entries from two days ago to today, the current query will also include the entries from yesterdayday, because the range is `two days ago` to `today`. It should be `two days ago` to `24 hours after two days ago` instead.

Currently this will only work if the dates are exactly one day apart from each other.

Reply:
I am not following, `list_entries` calls `from` and `to` as the **same day** from we want to copy entries. It will only list entries for that particular day. Not seeing why it would copy any other day.

Reply:
Ah crap, I read `f_day` and `t_day` here. Sorry, I was talking utter nonsense!